Makefile

```diff
@@ -76,8 +77,8 @@ SHADER_OBJS = \
 	$(SRCDIR)/shaders/untextured_pixelshader.inl
 
 DEBUG := y
-CFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR)
-CXXFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR)
+CFLAGS += -I$(SRCDIR) -I$(THIRDPARTYDIR) -O3 -fno-strict-aliasing
```

Comment:
We should have the optimization flag be dependent on `DEBUG` so they're not fighting one another (I have DEBUG default as 99% of the time I'm developing tests when building)

Reply:
Done